From e1d5b746787647c9b566f9ad96417d579308e108 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sun, 20 May 2018 18:59:39 -0700
Subject: [PATCH] Do not use root user/group during install

OE has its own way of using fake root using pseudo

Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 8e5120f..bbb1c38 100755
--- a/Makefile
+++ b/Makefile
@@ -39,7 +39,7 @@ clean:
 	
 install: rel/$(PLATFORM)/analyzer $(DESTDIR)$(BINDIR) $(DESTDIR)$(CONFDIR)
 	for i in rel/$(PLATFORM)/* util/* */*_monitor; do f=`basename $$i`; rm -f $(DESTDIR)$(BINDIR)/$$f; cp $$i $(DESTDIR)$(BINDIR); done
-	for i in config/*; do install -g $(ROOT_GID) -o $(ROOT_UID) -m $(ROOT_MOD) $$i $(DESTDIR)$(CONFDIR); done
+	for i in config/*; do install -m 0644  $$i $(DESTDIR)$(CONFDIR); done
 	-killall -9 actisense-serial n2kd socketcan-writer || echo 'No running processes killed'
 
 zip:
-- 
2.17.0

